commercial/app/model/merchandise/events/Eventbrite.scala

```diff
@@ -199,21 +122,28 @@ object Eventbrite extends ExecutionContexts with Logging {
     }
   }
 
-  trait EBTicketHandler {
-    def tickets: Seq[EBTicket]
+  trait TicketHandler {
+    def tickets: Seq[Ticket]
+
+    lazy val displayPriceRange = {
```

Comment:
I believe that this is a trait/instantiation thing wherein, but I can't actually remember a good reason for it. I'll dig in and get back to you!